cmd/contour/serve.go

```diff
@@ -155,6 +155,9 @@ func registerServe(app *kingpin.Application) (*kingpin.CmdClause, *serveContext)
 
 	serve.Flag("debug", "Enable debug logging.").Short('d').BoolVar(&ctx.Config.Debug)
 	serve.Flag("kubernetes-debug", "Enable Kubernetes client debug logging with log level.").PlaceHolder("<log level>").UintVar(&ctx.KubernetesDebug)
+
+	serve.Flag("gateway-class-name", "Contour GatewayClass name.").PlaceHolder("<api_group>/<namespace>/contour").StringVar(&ctx.gatewayClassName)
```

Comment:
maybe this should be `gateway-controller-name` or `gatewayclass-controller-name` or even `gatewayapi-controller-name`

currently this reads to me like the name of a `GatewayClass` resource

Reply:
Per https://github.com/projectcontour/contour/pull/3659#discussion_r637055343, I moved this to the config file and renamed to `ControllerName`.